### DISCUSSION TITLE
German tranlation broken

### DISCUSSION THREAD

**Discussion by klausladner** [xournalpp/xournalpp#5996] [General]
In previous versions German translation was fine. Nowadays, even the top leven menu items are still in English. The Crowdin project tells that that 98% are done, but only 54% are approved and the website ttells contribution is welcome for other languages than German. 
Nevertheless, can I contribute somehow on joining Crowdin? Or is approval a matter of admin stuff? Can I otherwise kind of wake up the admins?

**Comment by rolandlo** [comment]
Are you using the Windows version? If so, have you read the release notes on the release page?

**Reply by klausladner** [reply]
Yes, I am on Windows and meanwhile I found the hint you are talking about. It worked. Thanks.

**Comment by rolandlo** [comment]
> Nevertheless, can I contribute somehow on joining Crowdin? Or is approval a matter of admin stuff? Can I otherwise kind of wake up the admins?

By the way everyone can join the Xournal++ Crowdin project. No approval is needed. The German translations are usually quite up to date though, while languages with fewer users are behind.
